### PR TITLE
REL: change versions to 0.2.0

### PR DESCRIPTION
**Pull request by josenavas**
Preparation for the release, change version numbers from 0.1.0-dev to 0.2.0

### CODE REVIEW

**Comment by josenavas** [comment]
@antgonza @ElDeveloper quick review of this so I can prepare the release?

**Comment by antgonza** [comment]
:+1:

**Comment by josenavas** [comment]
We're testing that the system works installing it from scratch. Do not merge until we're sure about that, so this is the last commit in the release.

**Comment by ElDeveloper** [comment]
Can you also modify the ChangeLog to reflect the new development cycle?

**Comment by josenavas** [comment]
Sure! Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Qiita changelog
 
-
-Qiita 0.1.0-dev (changes since Qiita 0.1.0 go here)
+Qiita 0.2.0-dev (changes since Qiita 0.2.0 go here)
 ---------------------------------------------------
 
+Qiita 0.2.0
```
Comment (ElDeveloper) [inline CHANGELOG.md]: Can you change this to read `Version 0.2.0 (2015-08-25`? See the previous title for an example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # Qiita changelog
 
-
-Qiita 0.1.0-dev (changes since Qiita 0.1.0 go here)
+Qiita 0.2.0-dev (changes since Qiita 0.2.0 go here)
```
Comment (ElDeveloper) [inline CHANGELOG.md]: Can you remove this line? This should be added as part of #1434

**Comment by ElDeveloper** [comment]
Two comments, then we should be good to merge this.

**Pull request merged**